web/client/components/geostory/contents/ContentToolbar.jsx

```diff
@@ -100,8 +100,8 @@ export default function ContentToolbar({
                     noTooltipWhenDisabled: true
                 }}
                 buttons={tools
-                    .filter((id) => toolButtons[id])
-                    .map(id => toolButtons[id](props))}/>
+                    .filter((tool) => tool?.id && toolButtons[tool.id] || toolButtons[tool])
```

Comment:
Can you document more the "tools" property documentation to document this alternative (object).